src/index.js

```diff
@@ -1,7 +1,12 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import {BrowserRouter as Router} from 'react-router-dom';
 import './index.css';
 import App from './App';
 import 'bootstrap/dist/css/bootstrap.css';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+ReactDOM.render(
+        <Router>
+            <App />
+        </Router>, document.getElementById('root')
+    );
```

Comment:
It looks like your indentation is off here, and so if the formatting for your arguments. This is preferred:
```javascript
ReactDOM.render(
  <Router>
    <App />
  </Router>,
  document.getElementById('root')
);
```